_sass/layouts/_blog.scss

```diff
@@ -6,9 +6,10 @@
   height: 100%;
   font-size: 1.2em;
   text-align: center;
-  color: $header-text-color;
-  background: no-repeat $header-background-color;
-  background-size: $feature-image-size;
+  color: var(--gray8);
+  background: no-repeat var(--blue);
+  backdrop-filter: brightness(0.5);
```

Comment:
What does this new attribute do?

Reply:
![Screenshot from 2021-04-23 16-21-38](https://user-images.githubusercontent.com/13279154/115920040-05686a00-a450-11eb-8e15-63084fc76ec1.png)

Just replace the variables. I guess $header-background-color was similar to blue with reduced brightness.